siriuspy/siriuspy/devices/bpm.py

```diff
@@ -57,7 +57,7 @@ class BPM(_Device):
         'ACQTriggerDataHyst-SP', 'ACQTriggerDataHyst-RB',
         'SwTagEn-Sel', 'SwTagEn-Sts',
         'TbtTagEn-Sel', 'TbtTagEn-Sts',
```

Comment:
And perhaps change "Tbt" to "TbT" as well.

Reply:
Nice, let's do it!

Reply:
done!